docs/guides/mysql/backup/kubestash/overview/index.md

```diff
@@ -0,0 +1,98 @@
+    parent: guides-mysql-backup-stashv2
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+{{< notice type="warning" message="Please install [KubeStash](https://kubestash.com/docs/latest/setup/install/kubestash/) to try this feature. Database backup with KubeStash is already included in the KubeDB license. So, you don't need a separate license for KubeStash." >}}
+
+# MySQL Backup & Restore Overview
+
+KubeDB also uses [KubeStash](https://kubestash.com) to backup and restore databases. KubeStash by AppsCode is a cloud native data backup and recovery solution for Kubernetes workloads and databases. KubeStash utilizes [restic](https://github.com/restic/restic) to securely backup stateful applications to any cloud or on-prem storage backends (for example, S3, GCS, Azure Blob storage, Minio, NetApp, Dell EMC etc.).
+
+<figure align="center">
+  <img alt="KubeDB + KubeStash" src="/docs/guides/mysql/backup/kubestash/overview/images/kubedb_plus_kubestash.svg">
+<figcaption align="center">Fig: Backup KubeDB Databases Using KubeStash</figcaption>
+</figure>
+
+## How Backup Works
+
+The following diagram shows how KubeStash takes backup of a `MySQL` database. Open the image in a new tab to see the enlarged version.
+
+<figure align="center">
+  <img alt="MySQL Backup Overview" src="/docs/guides/mysql/backup/kubestash/overview/images/backup_overview.svg">
+  <figcaption align="center">Fig: MySQL Backup Overview</figcaption>
+</figure>
+
+The backup process consists of the following steps:
+
+1. At first, a user creates a `Secret`. This secret holds the credentials to access the backend where the backed up data will be stored.
+
+2. Then, she creates a `BackupStorage` custom resource that specifies the backend information, along with the `Secret` containing the credentials needed to access the backend.
+
+3. KubeStash operator watches for `BackupStorage` custom resources. When it finds a `BackupStorage` object, it initializes the `BackupStorage` by uploading the `metadata.yaml` file to the specified backend.
+
+4. Next, she creates a `BackupConfiguration` custom resource that specifies the target database, addon information (including backup tasks), backup schedules, storage backends for storing the backup data, and other additional settings.
+
+5. KubeStash operator watches for `BackupConfiguration` objects.
+
+6. Once the KubeStash operator finds a `BackupConfiguration` object, it creates `Repository` with the information specified in the `BackupConfiguration`.
+
+7. KubeStash operator watches for `Repository` custom resources. When it finds the `Repository` object, it Initializes `Repository` by uploading `repository.yaml` file into the `spec.sessions[*].repositories[*].directory` path specified in `BackupConfiguration`.
+
+8. Then, it creates a `CronJob` for each session with the schedule specified in `BackupConfiguration` to trigger backup periodically.
```

Comment:
The Operator creates a `CronJob` for each session with the schedule specified in the corresponding `BackupConfiguration` to trigger backup periodically.